build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -18,64 +18,21 @@
 ###########################################################################
 # imxrt1020-evk_download.sh
```

Comment:
Please consider the suggestions for this commit description similar to the previous one (rtk download script changes)

Reply:
Done.